invenio_sipstore/models.py

```diff
@@ -71,24 +65,35 @@ class SIP(db.Model, Timestamp):
     agent = db.Column(JSONType, default=lambda: dict(), nullable=False)
     """Agent information regarding given SIP."""
 
+    archivable = db.Column(
```

Comment:
Is this flag necessary? perhaps we could reduce it to a single "state" flag (enum)?